toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -232,16 +237,30 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
 	}
 
 	if len(rpmConflicts) != 0 {
-		logger.Log.Error("RPM Conflicts with toolchain:")
-		for _, conflict := range rpmConflicts {
-			logger.Log.Errorf("--> %s", conflict)
+		if !allowToolchainRebuilds {
+			logger.Log.Infof("RPM conflicts with toolchain: ")
+			for _, conflict := range rpmConflicts {
+				logger.Log.Infof("--> %s", conflict)
+			}
+		} else {
+			logger.Log.Error("RPM conflicts with toolchain: ")
+			for _, conflict := range rpmConflicts {
+				logger.Log.Errorf("--> %s", conflict)
+			}
 		}
 	}
 
 	if len(srpmConflicts) != 0 {
-		logger.Log.Error("SRPM Conflicts with toolchain:")
-		for _, conflict := range srpmConflicts {
-			logger.Log.Errorf("--> %s", conflict)
+		if !allowToolchainRebuilds {
```

Comment:
Same as above

Reply:
Yeah, this is getting bloaty. Even more reason to consider the "logger in a variable" idea.